src/schema/nmdc.yaml

```
@@ -1813,6 +1815,14 @@ classes:
       - https://casrai.org/credit/
 
 enums:
+
+  StrandedOrentationEnum:
```

I think there is a typo in the name of this enum:

```suggestion
  StrandedOrientationEnum:
```